### ISSUE TITLE
UnhandledPromiseRejectionWarning: TypeError: name.replace is not a function

### ISSUE THREAD

**Issue by juli1** [kucherenko/jscpd#223]
**Describe the bug**
jscpd crashes when analysis the mesos project.

```
node_modules/jscpd/bin/jscpd -o /tmp/bla --min-lines 10 --reporters xml /home/julien/git/mesos                           21:50:40


(node:18567) UnhandledPromiseRejectionWarning: TypeError: name.replace is not a function
    at sanitizeLangName (/home/julien/node_modules/jscpd/build/tokenizer/prism.js:62:21)
    at /home/julien/node_modules/jscpd/build/tokenizer/prism.js:90:109
    at Array.forEach (<anonymous>)
    at createTokens (/home/julien/node_modules/jscpd/build/tokenizer/prism.js:90:27)
    at /home/julien/node_modules/jscpd/build/tokenizer/prism.js:60:137
    at Array.forEach (<anonymous>)
    at Object.tokenize (/home/julien/node_modules/jscpd/build/tokenizer/prism.js:60:83)
    at JSCPD.<anonymous> (/home/julien/node_modules/jscpd/build/jscpd.js:302:46)
    at step (/home/julien/node_modules/jscpd/build/jscpd.js:43:23)
    at Object.next (/home/julien/node_modules/jscpd/build/jscpd.js:24:53)
(node:18567) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
(node:18567) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

**To Reproduce**
1. Checkout the mesos project git repository `git clone https://github.com/apache/mesos.git`
2. Invoke `jscpd -o /tmp/foo --min-lines 10 --reporters xml /path/to/mesos`

**Expected behavior**
jscpd should return and produce the report.

**Desktop (please complete the following information):**
 - OS: linux ubuntu
 - OS Version [e.g. 22]
 - NodeJS Version 8.10.0
 - jscpd version 2.0.13

**Additional context**

**Comment by kucherenko** [comment]
thank you for the report, will fix it

**Comment by kucherenko** [comment]
Fixed in jscpd@2.0.14
